src/Particle/ParticleSet.h

```diff
@@ -160,6 +160,9 @@ class ParticleSet : public QMCTraits, public OhmmsElementBase, public PtclOnLatt
   ///distance tables that need to be updated by moving this ParticleSet
   std::vector<DistanceTableData*> DistTables;
 
+  /// Descriptions from distance table creation.  Same order as DistTables.
+  std::vector<std::string> distTableDescriptions;
```

Comment:
What is your consideration of keep one for each table? I'm naively thinking just one description for all tables.

Reply:
That requires changing how the description is generated.  I'm trying to make the minimal amount of changes to this code (for now).

Reply:
I see. I did a quick try and hit some issues. Basically, we need to improve the use of C++. AoS is the problem again.

Reply:
Agree. The AoS/SoA stuff should either totally go (simpler) or we should use some C++ trickery such that there are no switches and everything always works.